mapNumRange.md

```diff
@@ -0,0 +1,15 @@
+### mapNumRange
+
+Maps a number from one range to another range.
+
+Returns the mapped num from the in range to the out range. 
+
+```js
+const mapNumRange = (num, in_min, in_max, out_min, out_max) => {
+    return (num - in_min) * (out_max - out_min) / (in_max - in_min) + out_min;
+  }
```

Comment:
The whole snippet can be written in one line, no need for `return` as it's a single statement.